<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(48, 84, 44, 255), stop:0.0340909 rgba(14, 14, 14, 255), stop:0.971591 rgba(50, 12, 12, 255), stop:1 rgba(173, 24, 24, 255));

selection-background-color: rgb(155, 84, 255);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="fileName">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>$show_VFX_$seq_$shot_fx_$element</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Revision</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveas">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-left-color: rgb(0, 255, 0);
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.340909 rgba(38, 46, 43, 165), stop:0.988636 rgba(32, 123, 255, 255), stop:1 rgba(102, 255, 126, 255));
selection-background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(236, 70, 46, 255), stop:0.323864 rgba(14, 14, 14, 255), stop:1 rgba(50, 12, 12, 142));

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
